easybuild/easyconfigs/a/ABAQUS/ABAQUS-6.14.1-linux-x86_64.eb

```diff
@@ -0,0 +1,15 @@
+name = 'ABAQUS'
+version = '6.14.1-linux-x86_64'
+
+homepage = 'http://www.simulia.com/products/abaqus_fea.html'
+description = "Finite Element Analysis software for modeling, visualization and best-in-class implicit and explicit dynamics FEA."
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+sources = ['%s-%s.tgz' % (name, version.split('-')[0])]
+
+builddependencies = [('Java', '1.7.0_60')]
+
+preinstallopts = "export CHECK_DISK_SPACE=OFF && "
```

Comment:
it may not be, only exported variables are passed down into child shells:

```
[kehoste:~] $ FOO=bar /bin/bash -c "echo $FOO"

[kehoste:~] $ export FOO=bar && /bin/bash -c "echo $FOO"
bar
```